src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -65,7 +93,8 @@ trait EntityComponent {
 
     /** gets the given entity */
     def get(workspaceContext: SlickWorkspaceContext, entityType: String, entityName: String): ReadAction[Option[Entity]] = {
-      unmarshalEntities(joinOnAttributesAndRefs(findEntityByName(workspaceContext.workspaceId, entityType, entityName))).map(_.headOption)
+      val sql = sql"""#$baseEntityAndAttributeSql where e.#${quoteIdentifier("name")} = ${entityName} and e.#${quoteIdentifier("entity_type")} = ${entityType} and e.#${quoteIdentifier("workspace_id")} = ${workspaceContext.workspaceId.toString}""".as[EntityListResult]
```

Comment:
again, don't hold this PR for this comment, but: would love to see this turned into a function, something like:
```
def generateSql(whereClause: String): SqlActionBuilder = {
  // concatenate baseEntityeAndAttributeSql + whereClause
}

val whereClause = "...whatever..."
val sql = generateSql(whereClause).as[EntityListResult]
```

you could even put the `.as[EntityListResult]` in the reusable function, if the base sql will only be used in one use case